src/components/screens/PointOfInterest.js

```diff
@@ -210,11 +210,20 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
                 [category.iconName || MAP.DEFAULT_PIN]: 1,
                 iconName: category.iconName || MAP.DEFAULT_PIN,
                 id,
-                position: { latitude, longitude }
+                position: { latitude, longitude },
+                serviceName: title,
+                title
               }
             ]}
-            selectedMarker={id}
             mapStyle={styles.mapStyle}
+            interactivity={{
+              pitchEnabled: false,
+              rotateEnabled: false,
+              scrollEnabled: false,
+              zoomEnabled: false
+            }}
+            isMultipleMarkersMap={false}
+            selectedMarker={id}
```

Comment:
I don't know the exact cause of the problem, but in some POI categories (Gastronomie) the app crashes and I get this error.

![Simulator Screenshot - iPhone 16 Plus - 2025-05-13 at 09 29 26](https://github.com/user-attachments/assets/4dabcdbc-6432-4525-a06e-7a26e2f98754)